ocpmodels/common/relaxation/ase_utils.py

```diff
@@ -118,8 +118,8 @@ def __init__(
                 "config"
             ]
 
-            # Load the trainer based on the dataset used
-            if config["task"]["dataset"] == "trajectory_lmdb":
+            # Load the trainer based on the task description
+            if "forces" in config["task"]["description"]:
```

Comment:
Hmm not a fan of adding a dependency on description being specified for this to work.

We should probably deprecate the `description` field anyway; it's not use anywhere else.